WTraining/app/services/user.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const logger = require('../logger');
+const { User } = require('../models');
+
+const createUser = user =>
+  User.create(user)
+    .then(res => res)
```

Comment:
This `then` is not necessary, you can to retrieve the result in the `then` of the controller